tests/integration/serializers/base/associations/sideloading-model-test.js

```diff
@@ -91,7 +91,7 @@ test(`it can sideload a named has-many association`, function(assert) {
       name: 'Link',
       postIds: ['1']
     },
-    blogPosts: [
+    posts: [
```

Comment:
I think this is wrong. Top-level collections should use the model name - what if this `blogPost` is also related to another model?

Attribute keys should only affect their model